### PR TITLE
update change log

### PR DESCRIPTION
**Pull request by vhvb1989**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [ ] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [ ] Doxygen docs
- [ ] Unit tests
- [ ] No unwanted commits/changes
- [ ] Descriptive title/description
  - [ ] PR is single purpose
  - [ ] Related issue listed
- [ ] Comments in source
- [ ] No typos
- [ ] Update changelog
- [ ] Not work-in-progress
- [ ] External references or docs updated
- [ ] Self review of PR done
- [ ] Any breaking changes?

### CODE REVIEW

File: sdk/core/azure-core/CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 Thank you to our developer community members who helped to make Azure Core better with their contributions to this release:
 
-- Jørgen Lind _([GitHub](https://github.com/jorgen))_
```
Comment (antkmsft) [inline sdk/core/azure-core/CHANGELOG.md:17]: Unintended change?

**Review by danieljurek — APPROVED** [review]
LGTM. Let's confirm the release behavior before merging.

**Review by antkmsft — APPROVED** [review]
Are you making the same update for identity?

**Comment by antkmsft** [comment]
@vhvb1989, please also update Identity the same way and release it, but don't push to vcpkg.

**Pull request merged**